### ISSUE TITLE
Make camera parameters configurable

### ISSUE THREAD

**Issue by Indy2222** [DigitalExtinction/Game#68]
- Make all or most camera parameters configurable (e.g. movement speed, minimum and maximum zoom, etc.).
- Load & store the configuration to an appropriate path (e.g. XDG_CONFIG).
- Document the configuration format.
- Make sure that issues in the configuration are properly reported (logging and error and fall-backing to default?)

UI for the configuration is not part of the issue. Manual edits to the configuration file are good enough.

This relates to #69.

This is blocked by #243.
